platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -128,6 +135,11 @@ public void init() throws Exception {
         return export(Paths.get(exportDirectory));
     }
 
+    @Override
+    public Collection<Describable> getInfo() {
+        return listMigratableDescribables();
```

Comment:
Once we have blueprint configured to inject the service reference lists (see next comment), this method should only have to return the list of `DataMigratable` services.